lib/core/bytes.nit

```diff
@@ -260,6 +294,42 @@ redef class Text
 			b.append_ns_from(s.items, s.bytelen, from)
 		end
 	end
+
+	# Returns a new `Bytes` instance with the digest as content
+	#
+	#     assert "0B1F4D".hexdigest_to_bytes == [0x0Bu8, 0x1Fu8, 0x4Du8]
+	#
+	# REQUIRE: `self` is a valid hexdigest and hexdigest.length % 2 == 0
+	fun hexdigest_to_bytes: Bytes do
+		var b = bytes
+		var pos = 0
+		var max = bytelen
+		var ret = new Bytes.with_capacity(max / 2)
+		while pos < max do
+			var byt = 0x00u8
+			var tmp = b[pos]
+			assert tmp.is_valid_hexdigit
+			if tmp >= 0x30u8 and tmp <= 0x39u8 then
+				byt |= (tmp - 0x30u8) << 4
+			else if tmp >= 0x41u8 and tmp <= 0x46u8 then
+				byt |= (tmp - 0x37u8) << 4
+			else if tmp >= 0x61u8 and tmp <= 0x66u8 then
```

Comment:
This could be nice, admittedly.

Maybe some kind of lexical node could be added for such a common use, I'll open an issue for that